docs/es/docs/tutorial/query-params.md

```diff
@@ -3,7 +3,7 @@
 Cuando declaras otros parámetros de la función que no hacen parte de los parámetros de path estos se interpretan automáticamente como parámetros de "query".
 
 ```Python hl_lines="9"
-{!../../docs_src/query_params/tutorial001.py!}
+{*../../docs_src/query_params/tutorial001.py*}
```

Comment:
and add `hl[9]` here after `tutorial001.py` e.g. `{*../../docs_src/query_params/tutorial001.py hl[9] *}`